src/analyses/custom_bitvector_analysis.cpp

```diff
@@ -380,6 +380,18 @@ void custom_bitvector_domaint::transform(
             }
           }
         }
+        else if(identifier=="memcpy")
+        {
+          if(code_function_call.arguments().size()==3)
+          {
+            // we copy all tracked bits from op1 to op0
+            // we do not consider any bits attached to the size op2
+            dereference_exprt lhs_deref(code_function_call.arguments()[0]);
+            dereference_exprt rhs_deref(code_function_call.arguments()[1]);
+
+            assign_struct_rec(from, lhs_deref, rhs_deref, cba, ns);
+          }
```

Comment:
Comment on why we're ignoring the non-3-op case?

Reply:
memmove: yes.
memset is harder; this might remove taint, but you'd have to make sure that the entire data structure is overwritten.

Reply:
True, forget memset